lib/go/thrift/header_protocol.go

```diff
@@ -346,6 +329,13 @@ func (p *THeaderProtocol) Skip(ctx context.Context, fieldType TType) error {
 	return p.protocol.Skip(ctx, fieldType)
 }
 
+// SetTConfiguration implemtns TConfigurationSetter.
```

Comment:
... implem**e**nts ... (there is more than one instance of this typo)

Reply:
Thanks. Fixed.